tox.ini

```diff
@@ -51,10 +53,17 @@ deps =
     {alldeps,tests,coverage}: priority
     {alldeps,tests,coverage}: sphinx
 
-    py27-{alldeps,tests,coverage}-posix: pysqlite
+    py27-{alldeps,tests,coverage}-{posix,macos}: pysqlite
     py27-{alldeps,tests,coverage}: soappy
 
-    alldeps-windows: pypiwin32
+    windows: pypiwin32
+
+    {alldeps,tests}-macos: pyobjc-core
+    {alldeps,tests}-macos: pyobjc-framework-cfnetwork
```

Comment:
Also not really related to this branch, but it reminds me, we should be similarly discerning in `macosx_platform`, and not depend on all of `pyobjc`.